src/System.Private.CoreLib/Resources/Strings.resx

```diff
@@ -2228,7 +2228,7 @@
     <value>--- End of inner exception stack trace ---</value>
   </data>
   <data name="Exception_EndStackTraceFromPreviousThrow" xml:space="preserve">
-    <value>--- End of stack trace from previous location where exception was thrown ---</value>
+    <value>--- End of stack trace from previous location ---</value>
```

Comment:
I intend to return to my exception ToString prettying that was only partially completed in 3.0 and will take a look at these then.